client/core/bond.go

```diff
@@ -521,19 +799,17 @@ func (c *Core) PostBond(form *PostBondForm) (*PostBondResult, error) {
 			c.connMtx.Unlock()
 			return &PostBondResult{ /* no new bond */ }, nil
 		}
-		// dc.acct is now configured with encKey, privKey, and id for a new
-		// (unregistered) account.
 	}
 
 	// Ensure this DEX supports this asset for bond, and get the required
 	// confirmations and bond amount.
 	bondAsset, bondExpiry := dc.bondAsset(bondAssetID)
 	if bondAsset == nil {
-		return nil, newError(assetSupportErr, "dex server does not support fidelity bonds in asset %q", bondAssetSymbol)
+		return nil, newError(assetSupportErr, "dex host has not connected or does not support fidelity bonds in asset %q", bondAssetSymbol)
 	}
-	bondValidity := time.Duration(bondExpiry) * time.Second // bond lifetime
 
-	lockTime := time.Now().Add(2 * bondValidity).Truncate(time.Second) // default lockTime is double
+	bondValidity := minBondLifetime(c.net, int64(bondExpiry))
+	lockTime := time.Now().Add(bondValidity).Truncate(time.Second)
```

Comment:
Few lines below this, expireTime is wrongly set to `time.Now().Add(bondValidity)`. And I think that should be `earliestServerExpiry := time.Now().Add(bondExpiry)`? 

Or... if it's the actual time the server will expire the new bond, then it should be?
`expireTime := locktime.Add(-bondExpiry)`?

Reply:
Good catch. This `minBondLifetime` refactoring was incomplete here in `PostBond`.

Reply:
Sorry about breaking that.  Fixed that regression in https://github.com/decred/dcrdex/commit/01a356289c9a5ad134f471ba295699c8a44dbc87.
I had started my rotations going before I squashed in that bad `minBondLifetime` update.

Shall we rebase https://github.com/decred/dcrdex/pull/2025 on top of this PR now?  Or are there still questions regarding interface, types, semantics, etc.?